### PR TITLE
Fw 4505 use images create read ap is

### PR DESCRIPTION
**Pull request by sahil97**
### Description of Changes
- Updated _uppy_ to work with the new BE.
- Switched from AWS S3 upload to XHR upload.
- media upload now sends formData instead of json.
- Added extra fields to media upload namely `excludeFromGames`, `excludeFromKids` and an `isShared` flag.
- Added new endpoint to `api.media` to get upload endpoint

Please briefly explain the changes here.

### Checklist

- ~README / documentation has been updated if needed~
- [x] PR title / commit messages contain Jira ticket number if applicable
- ~Tests have been updated / created if applicable~

### Reviewers Should Do The Following:

- [x] Review the changes here~
- ~Pull the branch and test locally~

### Additional Notes

N/A

### CODE REVIEW

File: src/services/api/media.js [hunk 1]
```diff
@@ -18,31 +14,18 @@ const media = {
     const nextPage = response?.next
     return { ...response, nextPage, lastPage }
   },
+  getAudio: async ({ sitename, id }) =>
```
Comment (sahil97) [inline src/services/api/media.js:17]: Moved these 3 methods after the `get` method. no modification here just re-ordering.
Added an upload endpoint towards the end of the file.

File: src/services/api/media.js [hunk 2]
```diff
@@ -18,31 +14,18 @@ const media = {
...
+    apiBase.get(`${SITES}/${sitename}/${AUDIO_PATH}/${id}`).json(),
+  getImage: async ({ sitename, id }) =>
+    apiBase.get(`${SITES}/${sitename}/${IMAGE_PATH}/${id}`).json(),
+  getVideo: async ({ sitename, id }) =>
+    apiBase.get(`${SITES}/${sitename}/${VIDEO_PATH}/${id}`).json(),
   getMediaDocument: async ({ sitename, docId, docType }) =>
     apiBase.get(`${SITES}/${sitename}/${docType}/${docId}`).json(),
   getS3Url: async () =>
```
Comment (gmcauliffe) [inline src/services/api/media.js:25]: If we're no longer using the getS3Url service please feel free to remove it from here.

File: src/components/MediaBrowser/MediaBrowserData.js [hunk 3]
```diff
@@ -61,11 +61,10 @@ function MediaBrowserData({ docType }) {
     isInitialLoading,
   } = useInfiniteQuery(
```
Comment (gmcauliffe) [inline src/components/MediaBrowser/MediaBrowserData.js:62]: Not essential to do now, but we will want to move this into a dataHook, similar to useSearchLoader. Could be moved we re-implement  media search.

**Review by gmcauliffe — APPROVED** [review]
LGTM 👍

File: src/components/MediaBrowser/MediaBrowserData.js (hunk 3 above)
Comment (sahil97) [reply src/components/MediaBrowser/MediaBrowserData.js:62]: Sounds good. I'll add the note to the media-search ticket.

**Pull request merged**